### PR TITLE
Chat rooms, OnRoomRemovedFromLobby Callback

### PR DESCRIPTION
**Pull request by ah1053**
1. Added Chat rooms package [Source code](https://github.com/mobiledgex/unity-samples/pull/20) ,[Video](https://mobiledgex.slack.com/archives/CKVFN0XJ5/p1618952454363700)
[EDGECLOUD-3844](https://mobiledgex.atlassian.net/browse/EDGECLOUD-3844)

2. OnRoomRemovedFromLobby Callback added.
3. Fix typos observerable -> observable.
4. Disable Warning for playerTags with the internal encapsulation.

### CODE REVIEW

**Review by brucearmstrong — APPROVED** [review]
Looks good.

File: EdgeMultiplay/Scripts/DataContracts.cs [hunk 1]
```diff
@@ -679,6 +679,7 @@ public class Player
         /// helper instance variable for serializing and deserializing playerTags
         /// </summary>
         [DataMember (EmitDefaultValue = false)]
+#pragma warning disable 0649
```
Comment (franklin-huang-mobiledgex) [inline EdgeMultiplay/Scripts/DataContracts.cs:682]: What does this do?

**Review by franklin-huang-mobiledgex — APPROVED** [review]

File: EdgeMultiplay/Scripts/DataContracts.cs (hunk 1 above)
Comment (ah1053) [reply EdgeMultiplay/Scripts/DataContracts.cs:682]: Just to disable the Warning message "Player Tags is never used" in Unity Console Logs

**Review by lgarner — APPROVED** [review]
lgtm.

**Pull request merged**